src/components/Teams.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import Navigation from './Navigation';
+import './teams.css';
+
+function Teams() {
+  const teams = useSelector((state) => state.teams.teams.teams);
+  const loading = useSelector((state) => state.teams.loading);
+  const error = useSelector((state) => state.teams.error);
+
+  let content;
+
+  if (loading) {
+    content = (
+      <div className="loader">
+        <div data-glitch="Loading..." className="glitch">
+          Loading...
+        </div>
+      </div>
+    );
+  } else if (error) {
+    content = <div className="error">{error}</div>;
+  } else if (teams) {
+    content = (
+      <div className="box">
+        {teams.map((team) => (
+          <div className="box-2" key={team.idTeam}>
+            <div className="cover-div">
```

Comment:
- [ ] As requested by the previous reviewer you need to add information displaying some kind of metrics and context for the information the home page handles. Kindly add this.

- [ ] You have not added the search/filter feature as requested on the home page. Kindly add that.

  ![image](https://github.com/2004-okari/REACT_SOLO_CAPSTONE/assets/81468675/16d73ad6-0d9c-4665-9ad5-7b39070e3e5f)

Reply:
- Well resolved 👍🏽